Resources/Prototypes/Roles/Jobs/Security/brigmedic.yml

```
@@ -0,0 +1,53 @@
+  supervisors: job-supervisors-hos
+  canBeAntag: false
+  access:
+  - Security
+  - Brig
+  - Maintenance
+  - Service
+  - External
+  - Cryogenics
+  - Medical
+  special:
+  - !type:AddImplantSpecial
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)